methods/array_ops/src/pg_gp/array_ops.c

```diff
@@ -1929,3 +1971,126 @@ General_Array_to_Array(
     return pgarray;
 }
 
+/******* internal functions ********/
```

Comment:
Turns out we have a similar function in svd.cpp (svd_vec_mult_matrix). we can build something similar - it should end up in matrix_ops.cpp.